tests/Properties.hs

```diff
@@ -2112,6 +2120,10 @@ bp_tests =
     , testProperty "foldr1"      prop_foldr1BP
     , testProperty "foldr1'"     prop_foldr1BP'
     , testProperty "mapAccumL"   prop_mapAccumLBP
+    , testProperty "`foldr` is lazy" prop_Lfoldr_lazy
+    , testProperty "`foldr'` is strict" prop_Lfoldr'_strict
+    , testProperty "`foldr1` is lazy" prop_Lfoldr1_lazy
+    , testProperty "`foldr1'` is strict" prop_Lfoldr1'_strict
```

Comment:
Let's avoid backticks in test names, they are a pain to escape in `--pattern`, when filtering test suite by test name.

Reply:
Sure, I shall fix this.